src/test/java/lesson28/FirstGUITest.java

```diff
@@ -0,0 +1,71 @@
+package lesson28;
+
+import org.example.Session;
+import org.example.base.Wait;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class FirstGUITest {
+
+    @BeforeMethod
+    public void before() {
+    }
+    @Test
+    public void testExample1() {
+        Session.get().webdriver().get("https://demoqa.com/elements");
+        WebElement buttons = Session.get().webdriver().findElement(By.xpath("//div[@class='element-list collapse show']//li[@id='item-4']"));
+        buttons.click();
+        Wait.sleep(1000);
+        WebElement clickMe = Session.get().webdriver().findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[2]/div[2]/div[3]/button"));
+        clickMe.click();
+        Wait.sleep(1000);
+        WebElement message = Session.get().webdriver().findElement(By.xpath("//*[@id=\"dynamicClickMessage\"]"));
+        System.out.println("Message: " + message.getText());
```

Comment:
Here sould be `Assert.assertEquals(message.getText(), "You have done a dynamic click")`